### PR TITLE
identity server issuer by location

### PR DESCRIPTION
**Pull request by Thorrik58**
# Determine identity server location in service portal by location origin

## What

Using different identity server by location

## Why

Needed for prod environment

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/service-portal/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 export default {
   production: true,
   identityServer: {
-    IDENTITY_SERVER_ISSUER_URL: 'https://identity-server.dev01.devland.is',
+    IDENTITY_SERVER_ISSUER_URL:
+      window.location.origin === 'https://beta.minarsidur.island.is'
+        ? 'http://innskra.island.is'
```
Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts:6]: should it be https ?

Comment (Thorrik58) [reply apps/service-portal/src/environments/environment.prod.ts:6]: Not sure ill ask

Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts:6]:
```suggestion
        ? 'https://innskra.island.is'
```

**Review by obmagnusson — APPROVED** [review]
LGTM

**Pull request merged**